README.md

```diff
@@ -465,44 +481,48 @@ After running `ilab config init` your directories will look like the following o
    pip install vllm@git+https://github.com/opendatahub-io/vllm@v0.6.2
    ```
 
+   Then you can serve a Safetensors model:
+
    ```shell
    ilab model serve --model-path ~/.cache/instructlab/models/instructlab/granite-7b-lab
    ```
 
 ### 📣 Chat with the model (Optional)
 
-Because you're serving the model in one terminal window, you will have to create a new window and re-activate your Python virtual environment to run `ilab model chat` command:
+- Because you're serving the model in one terminal window, you will have to create a new window and re-activate your Python virtual environment to run `ilab model chat` command:
```

Comment:
One tip that would be useful here is temperature settings... I don't think we have fine grained temperature settings yet but for a temperature of one you pass `-gm` to ilab chat and it will set the temperature to 0 which makes it the most accurate.

see https://github.com/instructlab/instructlab/blob/0f9ba00931b66d822f8fcecf29b6bf391f9f9412/src/instructlab/model/chat.py#L642